charts/tekton-operator/templates/_helpers.tpl

```diff
@@ -103,9 +103,9 @@ tekton-operator
   {{- $image = .Values.operator.image.repository }}
 {{- else -}}
 {{- if .Values.openshift.enabled -}}
-  {{- $image = "ghcr.io/tektoncd/operator/github.com/tektoncd/operator/cmd/openshift/operator" -}}
+  {{- $image = "ghcr.io/tektoncd/operator/operator-303303c315a48490ba6517859ef65b77" -}}
```

Comment:
I'm curious to ask where this mirror address comes from and if there is a script for automatic updates?

Because so far I haven't seen the `release.yaml` list for version `v0.75`, logically speaking, it should be consistent with the mirror in the final release.yaml.

Reply:
Initially we had images on gcr under a specific repository gcr.io/tektoncd/operator
Now that images storage have been migrated to ghcr, I can see the images at this place for operator https://github.com/tektoncd/operator/pkgs/container/operator%2Foperator-303303c315a48490ba6517859ef65b77
thats why preserving that specific repo name with ghcr was not working